pypit/scripts/coadd_1dspec.py

```diff
@@ -95,6 +95,12 @@ def main(args, unit_test=False, path=''):
             if mtch_obj is None:
                 print("No object {:s} in file {:s}".format(iobj, fkey))
             elif len(mtch_obj) == 1:
+                #Check if optimal extraction is present in all  objects. If not, warn the user and set ex_value to 'box'.
```

Comment:
Is it whether the object is present or whether it is full of
NAN?  Or, is NAN what we get when optimal extraction
is not performed?